bundles/org.openhab.core.audio/src/main/java/org/openhab/core/audio/internal/javasound/AudioPlayer.java

```diff
@@ -60,7 +61,10 @@ public AudioPlayer(AudioStream audioStream) {
     @Override
     public void run() {
         SourceDataLine line;
-        AudioFormat audioFormat = convertAudioFormat(this.audioStream.getFormat());
+
+        org.openhab.core.audio.AudioFormat openhabAudioFormat = this.audioStream.getFormat();
```

Comment:
Just a nit-picking one. Avoid using `this` if possible.

```suggestion
        org.openhab.core.audio.AudioFormat openhabAudioFormat = audioStream.getFormat();
```